platform/zilliqa/model.go

```diff
@@ -12,3 +21,70 @@ type Tx struct {
 	Nonce          uint64 `json:"nonce"`
 	ReceiptSuccess bool   `json:"receiptSuccess"`
 }
+
+type ChainInfo struct {
+	NumTxBlocks   string `json:"NumTxBlocks"`
+}
+
+type TxRPC struct {
+	ID       string `json:"ID"`
+	Amount   string `json:"amount"`
+	GasLimit string `json:"gasLimit"`
+	GasPrice string `json:"gasPrice"`
+	Nonce    string `json:"nonce"`
+	Receipt  struct {
+		CumulativeGas string `json:"cumulative_gas"`
+		EpochNum      string `json:"epoch_num"`
+		Success       bool   `json:"success"`
+	} `json:"receipt"`
+	SenderPubKey string `json:"senderPubKey"`
+	Signature    string `json:"signature"`
+	ToAddr       string `json:"toAddr"`
+	Version      string `json:"version"`
+}
+
+func (t *TxRPC)toTx() Tx {
```

Comment:
can you implement this in the api.go as `NormalizeTx` ?

Reply:
+ add tests for normalizing func

Reply:
We have 3 models, `blockatlas.Tx`, `zilliqa.Tx` and `zilliqa.TxRPC`, I want to separate the concern, in `api.go` we only deal with first two.

I can add a test for converting `TxRPC` to `Tx`